src/renderer/rendered_window.rs

```diff
@@ -231,12 +229,7 @@ impl RenderedWindow {
                 self.scroll_t = (self.scroll_t + dt / settings.scroll_animation_length).min(1.0);
             }
 
-            self.current_scroll = ease(
-                ease_out_expo,
-                self.start_scroll,
-                self.scroll_destination,
-                self.scroll_t,
-            );
+            self.current_scroll = ease(ease_out_expo, self.start_scroll, 0.0, self.scroll_t);
```

Comment:
This change is confusing to me.  I dont understand why we are easing from start to 0.0

Reply:
Would changing the name from `position` to `offset` make it clearer?

Since it's simulating a spring, the zero position represents the resting position, or the target. So that means that the regular ease-based animation should do the same, start from the stretched-out position and reach the resting position.